consensus/src/dag/commit_signer.rs

```diff
@@ -0,0 +1,28 @@
+use crate::experimental::signing_phase::CommitSignerProvider;
+use aptos_crypto::bls12381;
+use aptos_types::validator_signer::ValidatorSigner;
+
+pub struct DagCommitSigner {
+    signer: ValidatorSigner,
+}
+
+impl DagCommitSigner {
+    pub fn new(signer: ValidatorSigner) -> Self {
```

Comment:
Why do we need to wrap ValidatorSigner?
So we can create a separate crate in the future?